### ISSUE TITLE
Accessibility improvements and keyboard navigation issues for the Editor

### ISSUE THREAD

**Issue by kdikov82** [telerik/kendo-react#1286]
Reported by client (Ticket ID: 1561246)

1. Alt + F10 should focus toolbar

2. NVDA narrator problems:
    -  Doesn't narrate the disabled buttons
    -  Narrates incorrect role of the toolbar controls (Ex: narrated checkbox for buttons)

3. Tab Order issues:
    - Few of the dropdowns do not open on Enter/spacebar key press
    - Few of the dropdowns get open, but unable to access the content from keyboard (color picker and background picker for example)
    - After closing dropdown with Esc key, keyboard navigation is not working properly

4. Aquatic mode	         
    - Toolbar is visible, but keyboard focus not is not visible on toolbar

5. Fast Pass
    - Aria-input-field-name, aria-required-children and button-name fast pass issues are present
